core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlan.java

```diff
@@ -38,6 +39,7 @@
     // a map of an operator ID to operator's outputs (a set of operator IDs)
     private LinkedHashMap<String, LinkedHashSet<String>> adjacencyList;
 
+    private HashMap<String, ISink> sinkOperatorMap;
```

Comment:
Is this sinkOperatorMap used anywhere besides initialization?

Reply:
No, the `sinkOperatorMap` can only be created after the logical plan is created.

Reply:
My question is, since `sinkOperatorMap` only appeared in line 42 and 51 in this file, and never used at other place (at least I did not find any place), what is its purpose? can we remove it?

Reply:
Yes, we can remove it.